Sources/ClientRuntime/Networking/Streaming/ByteStream.swift

```diff
@@ -9,7 +9,6 @@ import class Foundation.FileHandle
 
 public enum ByteStream {
     case buffer(ByteBuffer)
-    case reader(StreamReader)
```

Comment:
The stream reader case was unused and seems unnecessary since everything can be handled via the ByteBuffer. Please let me know if you think this needs to stay in though.

Reply:
I would push back on removing this as we don't the side effect yet. Let's keep this.

This can be cleanedup when doing streaming improvements.